src/Core/Worker.php

```diff
@@ -852,9 +843,9 @@ private static function findWorkerProcesses(&$passing_slow)
 		}
 
 		if ($found) {
-			$condition = "`id` IN (".substr(str_repeat("?, ", count($ids)), 0, -2).") AND `pid` = 0 AND NOT `done`";
+			$condition = "`id` IN (".substr(str_repeat("?, ", count($ids)), 0, -2).") AND `process` = 0 AND NOT `done`";
```

Comment:
Maybe name the column `process_id`?

Reply:
You're right.
As far as I can see, it's best practice in the DB-Structure to add the primary key name of the referenced table to the foreign key...